src/handler.rs

```diff
@@ -523,8 +524,22 @@ pub async fn handle(
                     Some(serde_json::to_string(&channel_list.channels)?)
                 }
                 LndCmd::AddPeer(peer) => {
-                    let result = client.add_peer(peer).await?;
-                    Some(serde_json::to_string(&result)?)
+                    if let Some(alias) = peer.alias.clone() {
```

Comment:
@Evanfeenstra please do you think there is a better way to handle this??

Because I cannot borrow `state` as mutable more than once at a time. So I had to do it this way